data/hai/hai missions.txt

```diff
@@ -1488,10 +1516,17 @@ mission "Pirate Troubles [3]"
 			
 			label negotiations
 			`	"Perhaps we will stop," the leader says with a smile. "Under one condition. We'd like a one time shipment of various Hai weapons and technology. The shipment should be as big as possible. I've seen the freighters those aliens have. They're massive. Load two or three up with the goods and we'll consider focusing our attention elsewhere."`
+			branch public
+				has "event: pirates respond to leaks"
 			choice
 				`	"Okay. I'll meet with the Hai and see what they can provide."`
 					goto tribute
 				`	"I'm afraid that I can't allow that."`
+					goto disallow
+			label public
+			choice
+				`	"I'm afraid that I can't allow that."`
```

Comment:
Ask the original mission writer? All I did was adapt what was there to accommodate the possibilities introduced by this PR.

Reply:
What do you mean ask the original mission writer (me)? The branch preventing the other choice from being made is added by this PR. If you want to adapt this mission to the current situation, then instead of making it so that the player has no choice but to attack the pirates, the response of the Hai should be changed from acquiescing to the demands to declining them.